src/main/java/com/fabriccommunity/spookytime/registry/SpookyBlocks.java

```diff
@@ -31,6 +36,8 @@
 	public static Block SPOOKIUM_ORE = register("spookium_ore", new OreBlock(FabricBlockSettings.copy(Blocks.IRON_ORE).build()), new Item.Settings().group(SpookyTime.GROUP).rarity(Rarity.EPIC));
 	public static Block SPOOKIUM_BLOCK = register("spookium_block", new Block(FabricBlockSettings.copy(Blocks.IRON_BLOCK).materialColor(MaterialColor.RED).build()), new Item.Settings().group(SpookyTime.GROUP).rarity(Rarity.EPIC));
 	;
+
+	public static final BlockEntityType<TinyPumpkinBlockEntity> TINY_PUMPKIN_ENTITY = register("tiny_pumpkin", TinyPumpkinBlockEntity::new, TINY_PUMPKIN);
```

Comment:
This variable name should be changed to `TINY_PUMPKIN_BLOCK_ENTITY` to be clearer that this is not an actual Entity. We will be having an actual Entity that likely uses this same model in the future, so keeping them distinct is quite important.

Reply:
Fixed.